skills/unraid/references/troubleshooting.md

```
@@ -0,0 +1,34 @@
+# Unraid API Troubleshooting Guide
+
+Common issues and solutions when working with the Unraid GraphQL API.
+
+## "Cannot query field" error
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)